src/snyk/snykOss/views/suggestion/ossSuggestionWebviewProvider.ts

```diff
@@ -35,6 +37,24 @@ export class OssSuggestionWebviewProvider extends WebviewProvider<OssIssueComman
     );
   }
 
+  async postLearnLessonMessage(vulnerability: OssIssueCommandArg): Promise<void> {
+    if (this.panel) {
```

Comment:
I'm assuming here that in this context, snykOss only shows us vulnerabilities - no licence issues. Is that correct?

Reply:
Actually no. It shows license issues as well. Probably the internal union type had to be named as "Issue" and not "Vulnerability".

Reply:
Would I distinguish between the two using: OssVulnerability.exploit? It's a string though no a boolean.
```
export type OssVulnerability = {
  id: string;
  license?: string;
  identifiers?: Identifiers;
  title: string;
  description: string;
  language: string;
  packageManager: string;
  packageName: string;
  severity: OssSeverity;
  name: string;
  version: string;
  exploit?: string;

  CVSSv3?: string;
  cvssScore?: string;

  fixedIn?: Array<string>;
  from: Array<string>;
  upgradePath: Array<string>;
  isPatchable: boolean;
  isUpgradable: boolean;
};
```

Reply:
vulnerability.license should be the switch between vuln and license